### ISSUE TITLE
Content Review

### ISSUE THREAD

**Issue by JunyiSun** [OpenLiberty/draft-guide-microprofile-config#26]
Review the content of the guide, check if:
- [x] The code meets the requirements
- [x] The README.adoc meets the requirements

Thank you.

**Comment by yeekangc** [comment]
Content review is for a thorough review of everything we have for a guide to ensure that it meets the standards and quality that we want or the expectations that we have in a guide for Open Liberty.

**Comment by janamanoharan** [comment]
Just finished the guide, content wise it looks good and I got a better understanding of the Config feature. I was working with mpConfig earlier this week, so I was able to use that to finsih this guide as well.
I did find a couple typos ( or I think they are typos):
1. I think this line meant to say higher ordinal value
![image](https://user-images.githubusercontent.com/18078328/35421212-d33b94d4-020f-11e8-9874-eafdb68c887c.png)

2. The first sentence here seems a bit off. Is it suppose to say "Note that three default config sources mentioned above are static and fixed on application starting, so the properties within them <strong>cannot be modified</strong> while the server is running."
![image](https://user-images.githubusercontent.com/18078328/35421250-056c3f80-0210-11e8-90a9-2dcd640a9ca8.png)
